cmd/ctr/commands/run/run_windows.go

```diff
@@ -29,6 +29,13 @@ import (
 	"github.com/urfave/cli"
 )
 
+func init() {
+	ContainerFlags = append(ContainerFlags, cli.BoolFlag{
+		Name:  "hyperv",
+		Usage: "enable hyperv isolation for a windows container",
```

Comment:
Nit: `enable Hyper-V isolation`

Reply:
I'm not sure about the casing rules here, but probably should be `Windows` too, for UI.